### PR TITLE
fix Chat input overlaps the last message on IPhone 14

### LINKED ISSUE #16331: `Jump to` overlaps the last message in chat view in case of scrolling back and forth
# Bug Report
Follow-up on design call from 19/06/23.

## Problem 
`Jump to button` should not overlap the last message in chat

photo_2023-06-20 14 08 40 jpeg 2023-06-20 14-09-25

Designs:
https://www.figma.com/file/wA8Epdki2OWa8Vr067PCNQ/Composer-for-Mobile?type=design&node-id=3101-347945&t=Yspiy1M9ZTNBqu6s-4

### Reproduction

1) Open Status
2) Send message 

### Additional Information

- Status version: nightly 20/06/23
- Operating System: Android, iOS

### LINKED ISSUE #17112: Chat input overlaps the last message on IPhone 14
# Bug Report

## Problem 

https://github.com/status-im/status-mobile/assets/4557972/e02f936b-773b-47c5-b4b4-cfe05e04213f

- Status version: nightly 25/08/23
- Operating System: iOS 16.6 (iPhone 14 Pro) 
Can't be reproducible on IPhone 12  mini, Iphone 10. 

[comment]: # (Please, add logs/notes if necessary)

### PR DESCRIPTION
**Pull request by Parveshdhull**
fixes https://github.com/status-im/status-mobile/issues/17112
fixes https://github.com/status-im/status-mobile/issues/16331

status: ready

### CODE REVIEW

File: src/status_im2/contexts/chat/messages/list/view.cljs [hunk 1]
```diff
@@ -129,14 +124,11 @@
   [insets able-to-send-message?]
   [rn/view
    {:background-color (colors/theme-colors colors/white colors/neutral-95)
-    :margin-bottom    (when platform/ios? (- style/overscroll-cover-height))
```
Comment (Parveshdhull) [inline src/status_im2/contexts/chat/messages/list/view.cljs:132]: weird hack

Comment (Parveshdhull) [inline src/status_im2/contexts/chat/messages/list/view.cljs:132]: 🤔

Comment (Parveshdhull) [inline src/status_im2/contexts/chat/messages/list/view.cljs:132]: weird hack 🤔

Comment (flexsurfer) [reply src/status_im2/contexts/chat/messages/list/view.cljs:132]: yeah composer is full of weird hacks :(

**Review by flexsurfer — APPROVED** [review]

**Review by OmarBasem — COMMENTED** [review]
I don't know about these couple of files, so I will leave it to @alwx to approve, but generally looks good to me

**Comment by churik** [comment]
Wow, it looks great! 
I'll re-run e2e now(have to be more stable), no issues from my side - I'll add the label `Tested-OK` once I'll review e2e tests, meanwhile, I believe this requires design review.
@Francesca-G can you help?

**Review by Francesca-G — APPROVED** [review]
Looks good to me ✨

**Review by alwx — APPROVED** [review]

**Pull request merged**